README.md

```diff
@@ -19,6 +19,8 @@ It's mighty as it:
 - is **unopinionated** so that you can customize it to your exact needs
 - has a **growing community** and is used by [Facebook](https://code.facebook.com/posts/879890885467584/improving-css-quality-at-facebook-and-beyond/), [GitHub](https://github.com/primer/stylelint-config-primer) and [WordPress](https://github.com/ntwb/stylelint-config-wordpress/)
 
+Minimum NodeJs version supported : v10.13.0
```

Comment:
We can remove this line as using the `engine` field in the `package.json` will suffice. It's the standardised way of communicating a package's compatibility and will produce a warning on the CLI when `npm i` is used, e.g.:

```
npm WARN notsup Unsupported engine for stylelint@13.4.0: wanted: {"node":">=10.13.0"} (current: {"node":"8.17.0","npm":"6.13.4"})
```